legate/driver/command.py

```diff
@@ -395,6 +386,43 @@ def cmd_user_opts(
     cmd_log_levels,
     cmd_log_file,
     cmd_eager_alloc,
-    # Append user flags so they can override whatever we provided
-    cmd_user_opts,
+)
+
+CMD_PARTS_LEGION = (
+    (
+        cmd_bind,
+        cmd_rlwrap,
+        cmd_gdb,
+        cmd_cuda_gdb,
+        cmd_nvprof,
+        cmd_nsys,
+        # Add memcheck right before the binary
+        cmd_memcheck,
+        # Now we're ready to build the actual command to run
+        cmd_legion,
+        # This has to go before script name
+        cmd_nocr,
+        cmd_module,
+        cmd_processor,
+        cmd_kthreads,
```

Comment:
Is the force-kthreads option not compatible with canonical python mode?

Reply:
Good catch! It should be fixed by https://github.com/nv-legate/legate.core/pull/539/commits/3f89270fb8c5471737ccda8d618f81f107088471